src/controller/DonationsController.ts

```diff
@@ -0,0 +1,38 @@
+import { NextFunction, Request, Response } from "express";
+import Stripe from 'stripe';
+
+export class DonationsController {
+  async create(request: Request, _response: Response, _next: NextFunction) {
+    const {
+      amountUsd,
+    } = request.body;
+
+    const numberOfPizzas = Math.ceil(amountUsd / 20);
+    try {
+      const stripe = new Stripe(process.env.STRIPE_SECRET_KEY, {
+        apiVersion: '2020-08-27',
```

Comment:
Any reason we're not using `2020-10-24` ?

Reply:
Nope. It wasn't out when I did the initial exploratory work, and I hadn't checked to see if there was a new version yet. Will make the change!

Reply:
Oh I didn't check if it was out now so maybe this is a noop

Reply:
It appears that 2020-08-27 is the most recent version.